backend/app/contact/routes.py

```diff
@@ -1,8 +1,45 @@
-from flask import jsonify, abort
+from flask import jsonify, abort, request
 from app import db
 from app.models import Contact
 from . import contact
 
+# create a contact
+@contact.route("", methods=["POST"])
+def create_contact():
+    data = request.get_json(force=True)
+    name = data.get("name")
+    email = data.get("email")
+    secondary_email = data.get("secondary_email")
+    cellphone = data.get("cellphone")
+    role = data.get("role")
+    organization = data.get("organization")
+    neighbourhood = data.get("neighbourhood")
+    projects = data.get("projects")
+
+    if (
+        # all non-nullable attributes and relationships
+        name == ""
+        or email == ""
+        or cellphone == ""
+        or organization == ""
+        or projects == None
+    ):
+        abort(400, "Indicated fields cannot be empty")
+    new_contact = Contact(
+        name=name,
+        email=email,
+        secondary_email=secondary_email,
+        cellphone=cellphone,
+        role=role,
+        organization=organization,
+        neighbourhood=neighbourhood,
+        projects=projects,
+    )
+
+    db.session.add(new_contact)
+    db.session.commit()
```

Comment:
@dgiltz why is this here? There is no need for this to be here as you have a separate pr for the create a contact route.

Reply:
Sorry, problems with merging the code and my editor caused a lot of extraneous stuff to slip through